input/pagecontent/5_testplan.md

```diff
@@ -1,8 +1,69 @@
+PIXm specifies a query transaction between two actors.  The transaction between actors specify semantics of the data exchanged.  The PDQm test plan focuses on these semantics and on the expected actions on the server-side actor.
 
-Also, there is a peer-to-peer test definition "ITI-83_(PIXm)_Search_Read" used at Connectathons. This is coordinated for those at IHE Connectathon thru their login to Gazelle Test Management (EU or NA Connectathon).
\ No newline at end of file
+## High-level Test Scope
+
+### ITI-83 Patient Identifiers Cross-reference Query for Mobile 
+
+* PIX Consumer initiates ITI-83 queries for cross-referenced patients
+* PIX Manager responds to queries, as appropriate
+
+### ITI-104 Mobile Patient Identifier Cross-reference Feed
+
+* PIX Source initiates ITI-104 feed for:
+  * Add patient
+  * Revise patient
+  * Resolve patient duplicates
+  * Remove patient 
+* PIX Manager responds according to expected actions in ITI-104 for
+  * Add patient
+  * Revise patient
+  * Resolve patient duplicates
+  * Remove patient (if supported)
+
+### Options
+
+* none
+
+## Unit Test Procedure (Conformance Testing)
+
+Unit testing this context entails testing a SUT with a simulator or validator tool.  A simulator is an implementation of an actor that is designed specifically to test the opposite pair actor. The simulator might be a reference implementation or may be a specially designed test-bench.  Often, when a reference implementation is used, the negative tests are harder to simulate. A validator is an implementation that can check conformance. A validator may be a simulator, but may also be a standalone tool used to validate only a message encoding. Some reference implementations may be able to validate to a StructureDefinition profile, but often these do not include sufficient constraints given the overall actor conformance criteria. 
+
+### Available tools for PIXm unit testing
+
+#### Gazelle PatientManager - Simulator 
+
+* Provider: INRIA (Rennes, France), KEREVAL (https://www.kereval.com/)
+* Gazelle Patient Manager online: https://gazelle.ihe.net/PatientManager/home.seam
+* User Manual:  https://gazelle.ihe.net/gazelle-documentation/Patient-Manager/user.html
+* Tool support: https://gazelle.ihe.net/jira/projects/PAM
+* Actors (options) tested:  
+  * PIX Consumer, PIX Manager for ITI-83
+  * (future) PIX Source, PIX Manager for ITI-104
+
+#### Gazelle External Validation Service (aka "EVS Client") - Validator
+
+* Provider:  INRIA (Rennes, France), KEREVAL (https://www.kereval.com/), and Mallinckrodt Institute of Radiology (Saint Louis, USA) 
+* Tool location: https://gazelle.ihe.net/EVSClient/home.seam
+* Documentation: https://gazelle.ihe.net/content/evsfhirvalidation
+* Scope of testing: validation using StructureDefinitions for PIXm
+* Other notes: StructureDefinitions available in EVSClient validation are published by IHE here: https://github.com/IHE/fhir/tree/master/StructureDefinition
```

Comment:
I would suggest to remove the line 'StructureDefinitions available in EVSClient', now the StructureDefintion are available in this  implementation guide and I would propose to remove the old ones from https://github.com/IHE/fhir/tree/master/StructureDefinition and let Kereval handle the integration of this ig in EVSClient

Reply:
I thought about this, too.

We should discuss this, I think.    Right now, it's easy for me to tell the kereval people how to configure EVSClient -- just pull all the StructureDefinitions from the single place in gihub under IHE/fhir.

When we move to IGs, do I have to say, "get the MHD StructureDefinitions here, and the PIXm StructureDefinitions there, etc, and then get all of the rest of them from github under IHE/fhir????

Reply:
agree

Reply:
add link to npm package.